packages/react/src/components/select/utils.ts

```diff
@@ -18,6 +18,8 @@ export type OptionIterator = (
   groupIndex: number,
 ) => Option | undefined;
 
+export const DROPDOWN_MENU_ITEM_HEIGHT = 52;
```

Comment:
"The capitalized version is used for declaring constants in most programming languages. A constant is a data item whose value doesn't change throughout the life of a program."

This is normal practise also in HDS. Constants do not change.

It does not have "suffix", like "52px" because the number is needed for calculations.